src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingComment.java

```diff
@@ -24,10 +24,25 @@ void method2(long ms /* we should ignore this */) {
     }
 
     /**
-     * comment with trailing space
+     * comment with trailing space.
      */
     final static public String NAME="Some Name"; // NOI18N
     final static public String NAME2="Some Name"; /*NOI18N*/
     String NAME3="Some Name"; /*NOI18N
 */
+    void method3() {
+        /* violation on this block */
+        // violation here
```

Comment:
why violations are here ?
there is no code before comment on the same line.

Reply:
Sir, the violation here is when the configuration `format = NOT FOUND` . In default case, there is no violation. Should I make a file and put these examples there, as we are any way going to use BDD, and each test we have separate files?

Reply:
ok, I hope we will do BDD very soon.